target/linux/ramips/dts/mt7621_viettel_vap-120wd.dts

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "viettel,vap-120wd", "mediatek,mt7621-soc";
+	model = "Viettel VAP-120WD";
+
+	aliases {
+		led-boot = &led_status_blue_north;
+		led-failsafe = &led_status_red_east;
+		led-running = &led_status_blue_north;
+		led-upgrade = &led_status_red_east;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_blue_east: led@3 {
```

Comment:
`gpio-leds` doesn't use `reg` property and `-` should be used instead of `@`.

https://elixir.bootlin.com/linux/v6.6.52/source/Documentation/devicetree/bindings/leds/leds-gpio.yaml#L49